src/components/form/select/index.js

```diff
@@ -3,17 +3,32 @@ import cn from 'classnames';
 
 import './styles.scss';
 
-const Select = ({ register, options, name, error, placeholder, className = '', ...rest }) => (
+const Select = ({
```

Comment:
Bear in mind that select/option doesn't allow for customization like the icons inside each option.